### PR TITLE
wrap content plus changed some sizes and margins

### PR DESCRIPTION
**Pull request by michaone007**

### CODE REVIEW

File: app/src/main/res/layout/activity_trivia_game.xml [hunk 1]
```diff
@@ -336,40 +336,40 @@
 
     <RelativeLayout
         android:id="@+id/candyBox"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
```
Comment (zakiva) [inline app/src/main/res/layout/activity_trivia_game.xml:343]: הפוך גוטה הפוך

File: app/src/main/res/layout/activity_trivia_game.xml [hunk 2]
```diff
@@ -336,40 +336,40 @@
...
-            android:layout_height="15dp"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
             android:layout_toRightOf="@id/candy"
             android:gravity="center"
             android:text="700"
             android:textColor="#030330"
-            android:textSize="12sp"
+            android:textSize="15sp"
             android:textStyle="bold" />
```
Comment (zakiva) [inline app/src/main/res/layout/activity_trivia_game.xml:360]: same for `candiesText`

**Comment by michaone007** [comment]
ready to merge

File: app/src/main/res/layout/activity_trivia_game.xml (hunk 2 above)
Comment (zakiva) [reply app/src/main/res/layout/activity_trivia_game.xml:360]: ?

**Comment by zakiva** [comment]
@michaone007 see my comment here: https://github.com/zakiva/Santa/pull/52/files#diff-02c8464db7bd6872ad94a7ac608d4220R360

**Pull request merged**